### PR TITLE
Use ContractInfoResponse from cosmwasm_std

### PR DESCRIPTION
**Pull request by webmaster128**
If I understand correctly, "due to non_exhaustive" really means _we have no way to construct it_. This is a feature initially because ContractInfoResponse instances are always provided to the contract developer and it should be hard to create them.

In https://github.com/CosmWasm/cosmwasm/pull/1177 a constructor was added sets the two fields which have no default and use default values for the rest. This can be used now to remove the copied type.

### CODE REVIEW

File: packages/multi-test/src/wasm.rs [hunk 1]
```diff
@@ -25,20 +25,6 @@ use cosmwasm_std::testing::mock_wasmd_attr;
 
 use anyhow::{bail, Context, Result as AnyResult};
 
-// TODO: we should import this from cosmwasm-std, but cannot due to non_exhaustive so copy here
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
-pub struct ContractInfoResponse {
```
Comment (webmaster128) [inline packages/multi-test/src/wasm.rs:30]: This `pub` was never used, right? At least I did not find something.

Comment (ethanfrey) [reply packages/multi-test/src/wasm.rs:30]: It was not. Maybe pub for test cases?

**Review by ethanfrey — APPROVED** [review]
Good cleanup now we updated the code.

Thanks.

File: packages/multi-test/src/wasm.rs (hunk 1 above)
Comment (webmaster128) [reply packages/multi-test/src/wasm.rs:30]: Probably just copy/paste from the cosmwasm repo

**Pull request merged**